db/index.js

```diff
@@ -14,8 +14,12 @@ const Event = sequelize.define('Event', {
     unique: true,
     allowNull: false
   },
-  title: Sequelize.STRING,
-  summary: Sequelize.STRING,
+  title: {
+    type: Sequelize.STRING,
```

Comment:
I may need to add some extra regex stuff here to allow sequelize to deal with non english chars